ibm/service/power/ibm_pi_constants.go

```diff
@@ -351,6 +359,8 @@ const (
 	PIVolumeGroupID                   = "pi_volume_group_id"
 	PIVolumeGroupAction               = "pi_volume_group_action"
 	PIVolumeOnboardingID              = "pi_volume_onboarding_id"
+	PIVolumeProvisioning              = "pi_volume_provisioning"
+	PIVolumeProvisioningDone          = "pi_volume_provisioning_done"
```

Comment:
Those consts are not define properly. 
```
	helpers.PIVolumeProvisioning     = "creating"
	helpers.PIVolumeProvisioningDone = "available"
```
A state/status should be created instead. eg:
```
State_Creating = "creating"
State_Available = "available"
```